modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -1086,6 +1086,118 @@ describe('User CRUD tests', function () {
     });
   });
 
+  describe('Username Update', function () {
```

Comment:
This could actually be nice to have in a separate file. Probably not required.

Because the test file is already so huge.

Reply:
Totally agree \o/

Reply:
I split users tests to multiple files to make life easier: https://github.com/Trustroots/trustroots/pull/629

username changed tests should probably be either at `user-profile` tests or if there are lot of them, then their own `user-profile-username` tests.